app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ActiveRecord::Base
+  has_many :comments, foreign_key: :user_id
+  has_many :posts, foreign_key: :user_id
+  has_many :likes, foreign_key: :user_id
+
+  def three_recent_post
+    posts.order('created_at').limit(3)
```

Comment:
- [ ] The post should be in descending order.

`posts.order('created_at Desc').limit(3)`